incubator/hnc/cmd/manager/main.go

```diff
@@ -81,6 +82,7 @@ func main() {
 	flag.IntVar(&maxReconciles, "max-reconciles", 1, "Number of concurrent reconciles to perform.")
 	flag.IntVar(&qps, "apiserver-qps-throttle", 50, "The maximum QPS to the API server.")
 	flag.BoolVar(&stats.SuppressObjectTags, "suppress-object-tags", true, "If true, suppresses the kinds of object metrics to reduce metric cardinality.")
+	flag.IntVar(&port, "port", 443, "port for the server. defaulted to 443 if unspecified ")
```

Comment:
Use sentence case (eg capitalize the first word). Also say _which_ service this is for - e.g. the webhook. Maybe also change the flag name from `--port` to `--admission-webhook-port` or something like that (ditto for the internal variable).

Reply:
sg. Updated to `webhook-server-port` to match the controller-runtime description of the port.